recipes/label-studio-sdk/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "label-studio-sdk" %}
+{% set version = "0.0.32" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/label-studio-sdk-{{ version }}.tar.gz
+  sha256: 83ba94157f3849748f1c7b9d56a8f708db3c3f8dc124af4c9e7abf1eddc295c2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - lxml >=4.2.5
+    - requests >=2.22.0,<3
+    - pydantic <3,>1.7
+    - label-studio-tools >=0.0.1
+
+test:
+  imports:
+    - docs
```

Comment:
This looks like the docs are getting installed. You should patch the find_packages call in setup.py/pyproject.toml to omit it.

Reply:
I modified `setuptools.find_packages`, which was merged yesterday:

` packages=setuptools.find_packages(exclude=["docs"])`

Is this enough/correct? Do I need to wait for the pypi version to be updated before regenerating the recipe?